clients/lsp-volar.el

```diff
@@ -135,74 +111,11 @@ in the WORKSPACE-ROOT."
-  :activation-fn 'lsp-volar--activate-p
-  :priority 0
-  :multi-root nil
-  :add-on? t
-  :server-id 'volar-doc
-  :initialization-options (lambda () (ht-merge (lsp-configuration-section "typescript")
-                                               (ht ("languageFeatures" (ht-merge (ht ("documentHighlight" t))
-                                                                                 (ht ("documentLink" t))
-                                                                                 (ht ("codeLens" (ht ("showReferencesNotification" t))))
-                                                                                 (ht ("semanticTokens" t))
-                                                                                 (ht ("diagnostics" (ht ("getDocumentVersionRequest" nil))))
-                                                                                 (ht ("schemaRequestService" (ht ("getDocumentContentRequest" nil)))))))))
-  :initialized-fn (lambda (workspace)
-                    (with-lsp-workspace workspace
-                      (lsp--server-register-capability
-                       (lsp-make-registration
-                        :id "random-id"
-                        :method "workspace/didChangeWatchedFiles"
-                        :register-options? (lsp-make-did-change-watched-files-registration-options
-                                            :watchers
-                                            `[,(lsp-make-file-system-watcher :glob-pattern "**/*.js")
-                                              ,(lsp-make-file-system-watcher :glob-pattern "**/*.ts")
-                                              ,(lsp-make-file-system-watcher :glob-pattern "**/*.vue")
-                                              ,(lsp-make-file-system-watcher :glob-pattern "**/*.jsx")
-                                              ,(lsp-make-file-system-watcher :glob-pattern "**/*.tsx")
-                                              ,(lsp-make-file-system-watcher :glob-pattern "**/*.json")])))))
-  :download-server-fn (lambda (_client callback error-callback _update?)
-                        (lsp-package-ensure 'volar-language-server
-                                            callback error-callback))))
-
-(lsp-register-client
- (make-lsp-client
-  :new-connection (lsp-stdio-connection
-                   (lambda ()
-                     `(,(lsp-package-path 'volar-language-server) "--stdio")))
-  :activation-fn 'lsp-volar--activate-p
-  :priority 0
-  :multi-root nil
-  :add-on? t
-  :server-id 'volar-html
+  :server-id 'vue-semantic-server
   :initialization-options (lambda () (ht-merge (lsp-configuration-section "typescript")
-                                               (lsp-configuration-section "documentFeatures")))
+                                               (ht ("serverMode" 0)
+                                                   ("diagnosticMode" 1)
```

Comment:
Here are a wrong field: `diagnosticMode` -> `diagnosticModel`

Reply:
Thanks for your correction, I will fix it.